tf_trees/README.md

```diff
@@ -1,18 +1,47 @@
 # The Tree Ensemble Layer
 
-This repository contains a new *tree ensemble layer* which can be used anywhere in a neural network. We provide a low-level Tensorflow implementation along with a high-level Keras API.
+This repository contains a new *tree ensemble layer* (TEL) for neural networks. The layer is differentiable so SGD can be used to train the neural network (including TEL). The layer supports conditional computation for both training and inference, i.e., when updating/evaluating a certain node in the tree, only the samples that reach that node are used in computations (this is to be contrasted with the dense computations in neural networks). We provide a low-level Tensorflow implementation along with a high-level Keras API.
 
-The layer is differentiable so SGD can be used to train the neural network (including the tree layer). The layer supports conditional computation for training and inference: when updating/evaluating a certain node in the tree, only the samples that reach that node are used in computations (this is to be contrasted with the dense computations in neural networks).
+More details to be added soon.
+
+## Installation
+The installation instructions below assume that you have Python, TensorFlow, and GCC already installed.
```

Comment:
So since it is a custom kernel not included in the pre-built, don't you need to rebuild tensorflow with bazel for it to be picked up?

Reply:
I am not sure what the steps below do so that's why i am asking

Reply:
These instructions are following https://www.tensorflow.org/guide/create_op#compile_the_op_using_your_system_compiler_tensorflow_binary_installation 

Yeah it can also be installed using bazel. Should we add installation using bazel as well?

Reply:
I didn't know your way. Can you add a comment with bazel too - i think it is more common, but i might be wrong